lld/test/ELF/section-symbols.test

```diff
@@ -1,3 +1,4 @@
+# UNSUPPORTED: system-windows
```

Comment:
This is also upstream, so it needs an explanation and perhaps a Matter Labs wrapper.  Have pity upon a poor Windows-phobic compiler engineer who might someday have to deal with this file.